celestia/static/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>celestia-node-rs</title>
+        <script type="module" src="/js/run_node.js"></script>
+    </head>
+    <body>
+        <h1>Celestia Node</h1>
+        <h2>Network</h2>
+        <select id="network_id" class="config">
+            <option value="0">Arabica</option>
+            <option value="1">Mocha</option>
+            <option value="2">Private</option>
+        </select>
+        <h2>Bootnodes</h2>
+        <textarea id="bootnodes" class="config" cols=80 rows=10>
```

Comment:
may we get this wider? I don't really care about overflows but this way it can fit all bootstrap mas without splitting
```suggestion
        <textarea id="bootnodes" class="config" cols=220 rows=10>
```